src/libs/SubscriptionUtils.ts

```diff
@@ -1,9 +1,122 @@
 import {differenceInSeconds, fromUnixTime, isAfter, isBefore, parse as parseDate} from 'date-fns';
-import type {OnyxCollection, OnyxEntry} from 'react-native-onyx';
 import Onyx from 'react-native-onyx';
+import type {OnyxCollection, OnyxEntry} from 'react-native-onyx';
 import CONST from '@src/CONST';
+import type {OnyxValues} from '@src/ONYXKEYS';
 import ONYXKEYS from '@src/ONYXKEYS';
-import type {BillingGraceEndPeriod, Policy} from '@src/types/onyx';
+import type {BillingGraceEndPeriod, Fund, Policy} from '@src/types/onyx';
+
+const PAYMENT_STATUSES = {
+    POLICY_OWNER_WITH_AMOUNT_OWED: 'policy_owner_with_amount_owed',
+    POLICY_OWNER_WITH_AMOUNT_OWED_OVERDUE: 'policy_owner_with_amount_owed_overdue',
+    OWNER_OF_POLICY_UNDER_INVOICING: 'owner_of_policy_under_invoicing',
+    OWNER_OF_POLICY_UNDER_INVOICING_OVERDUE: 'owner_of_policy_under_invoicing_overdue',
+    BILLING_DISPUTE_PENDING: 'billing_dispute_pending',
+    CARD_AUTHENTICATION_REQUIRED: 'authentication_required',
+    INSUFFICIENT_FUNDS: 'insufficient_funds',
+    CARD_EXPIRED: 'expired_card',
+    CARD_EXPIRE_SOON: 'card_expire_soon',
+    RETRY_BILLING_SUCCESS: 'retry_billing_success',
+    RETRY_BILLING_ERROR: 'retry_billing_error',
+    GENERIC_API_ERROR: 'generic_api_error',
+};
```

Comment:
```suggestion
} as const;
```

Maybe? If it makes sense.